packages/linux/package.mk

```diff
@@ -89,6 +89,12 @@ post_patch() {
     sed -i -e "s|^CONFIG_ISCSI_IBFT_FIND=.*$|# CONFIG_ISCSI_IBFT_FIND is not set|" $PKG_BUILD/.config
     sed -i -e "s|^CONFIG_ISCSI_IBFT=.*$|# CONFIG_ISCSI_IBFT is not set|" $PKG_BUILD/.config
   fi
+
+  # disable lima/panfrost if libmali is configured
+  if [ "$OPENGLES" = libmali ]; then
+    sed -i -e "s|^CONFIG_DRM_LIMA=.*$|# CONFIG_DRM_LIMA is not set|" $PKG_BUILD/.config
```

Comment:
Can you use `sed -e "<prog>" -i <filename>` as that is how we typically do it (same for below). Thanks.

Reply:
now it doesn't match the similar stuff above it, but someone else can clean those if needed

Reply:
Thanks. Grrr... I'll fix those other sed entries one day, a mass change of all those right now might not be a good idea while we're also backporting stuff.